src/pages/components/alert.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import alertContent from '../../../locale/fi/alert.json';
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import { Alert, InlineAlert } from 'components/ExampleComponents';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+import Section from 'components/Section';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentExample from 'components/ComponentExample';
+import { defaultSuomifiTheme } from 'suomifi-ui-components';
```

Comment:
We prefer to have external dependencies imported first, then internals.
So moving this under the React's import would be nice.